Makefile

```diff
@@ -351,12 +394,149 @@ teamviewer: | $(TEAMVIEWER)
 
 $(NODE): | $(CURL) $(BASH)
 	$(CURL) -sL https://deb.nodesource.com/setup_current.x | sudo -E $(BASH) -
-	sudo apt update
+	sudo apt update -y
 	sudo apt install -y nodejs
 
 node: | $(NODE)
 
 $(NPM): | $(NODE)
 npm: | $(NPM)
 
+$(BRAVE): | $(CURL)
+	sudo apt install -y apt-transport-https curl gnupg
+	$(CURL) -s https://brave-browser-apt-release.s3.brave.com/brave-core.asc \
+		| sudo apt-key --keyring /etc/apt/trusted.gpg.d/brave-browser-release.gpg add -
+	echo "deb [arch=amd64] https://brave-browser-apt-release.s3.brave.com/ stable main" \
+		| sudo tee /etc/apt/sources.list.d/brave-browser-release.list
+	sudo apt update -y
+	sudo apt install -y brave-browser
+
+brave: | $(BRAVE)
+
+$(FIREFOX):
+	sudo apt install -y firefox
+
+firefox: | $(FIREFOX)
+
+$(DNSMASQ): | $(BASH) $(UFW)
+	$(BASH) -c '[ -f /.dockerenv ] || sudo $(SYSTEMCTL) disable --now systemd-resolved'
+	$(BASH) -c '[ -f /.dockerenv ] || sudo rm -f /etc/resolv.conf'
+	echo 'nameserver 127.0.0.1' | sudo tee    /etc/resolv.conf
+	echo 'nameserver 1.1.1.1'   | sudo tee -a /etc/resolv.conf
+	echo 'nameserver 9.9.9.9'   | sudo tee -a /etc/resolv.conf
+	sudo mkdir -p /etc/NetworkManager/conf.d
+	echo "[main]\ndns=none\nrc-manager=unmanaged" | sudo tee /etc/NetworkManager/conf.d/dnsmasq.conf
+	sudo service --status-all | awk '{ print $$4 }' | grep -qv network-manager || sudo service network-manager restart
```

Comment:
Together with Arjen, I observed this command is skipped when it shouldn't be skipped.